Makefile

```diff
@@ -55,15 +56,16 @@ CXXFLAGS = -Wno-delete-non-virtual-dtor -Wno-gnu-designator -Wno-tautological-co
 CFLAGS   = -no-integrated-as
 ASFLAGS  =
 else
-CC  ?= $(TOOLPREFIX)gcc
-CXX ?= $(TOOLPREFIX)g++
+CC  = $(TOOLPREFIX)gcc
+CXX = $(TOOLPREFIX)g++
 CXXFLAGS = -Wno-delete-non-virtual-dtor  -fno-pie -fno-pic
 CFLAGS   = -fno-pie -fno-pic
 ASFLAGS  = -Wa,--divide -fno-pie -fno-pic
 endif
 
 LD = $(TOOLPREFIX)ld
 NM = $(TOOLPREFIX)nm
+AR = $(TOOLPREFIX)ar
 OBJCOPY = $(TOOLPREFIX)objcopy
 STRIP = $(TOOLPREFIX)strip
```

Comment:
Could you keep these in alphabetical order? We're pretty bad at that in general, but I figure it it already is we might as well try to keep it